### ISSUE TITLE
ITS Anonymizer

### ISSUE THREAD

**Issue by lucasgautheron** [LAAC-LSCP/ChildRecordsData#133]
**Is your feature request related to a problem? Please describe.**

Integrate the its anonymizer @ https://github.com/HomeBankCode/ITS_annonymizer into the package.

**Describe the solution you'd like**

No GUI.

```
child-project anonymize /path/to/dataset --input-set lena --output-set anonymized-lena --replacements-json-dict replacements.json
```

 `--replacements-json-dict` would be the path to the json file that defines the replacements. By default it is https://github.com/HomeBankCode/ITS_annonymizer/blob/master/replacements_dict.json, and my understanding is that the GUI of the original code allows you to generate different json files based on the selected rules. We could expect people to modify this file by hand (or even to use the original GUI to generate it if they like..)

**Comment by lucasgautheron** [comment]
Actually we might want to name the options --set and --new-set rather than --input-set and --output-set to be consistent with https://laac-lscp.github.io/ChildRecordsData/TOOLS.html#rename-a-set-of-annotations

or we could use --input-set --output-set everywhere. Which do you think is the best ?

**Comment by lucasgautheron** [comment]
Closed by #134
